.circleci/config.yml

```diff
@@ -125,7 +128,10 @@ jobs:
 
       - run: bundle exec fastlane validate_docs
 
-  "Lint Source Code":
+      - store_artifacts:
+          path: /tmp/validated_docs
```

Comment:
Unfortunately CircleCI 2 doesn't let you use env variables here :/
(Correct @DanToml?)